nipype/algorithms/tests/test_confounds.py

```diff
@@ -16,3 +24,16 @@ def test_fd():
     yield assert_equal, np.allclose(ground_truth, np.loadtxt(res.outputs.out_file)), True
     yield assert_equal, np.abs(ground_truth.mean() - res.outputs.fd_average) < 1e-4, True
     rmtree(tempdir)
+
+@skipif(nonitime)
+def test_dvars():
+    tempdir = mkdtemp()
+    ground_truth = np.loadtxt(example_data('ds003_sub-01_mc.DVARS'))
+    dvars = ComputeDVARS(in_file=example_data('ds003_sub-01_mc.nii.gz'),
+                         in_mask=example_data('ds003_sub-01_mc_brainmask.nii.gz'),
+                         save_all = True)
+    os.chdir(tempdir)
+    res = dvars.run()
+
+    dv1 = np.loadtxt(res.outputs.out_std)
+    yield assert_equal, (np.abs(dv1 - ground_truth).sum()/ len(dv1)) < 0.05, True
```

Comment:
Honestly: the MSE I was getting was 0.048.

In detail: I run our implementation and @nicholst 's, stopping at every step and comparing outputs. Once I found out that they were exactly the same except for some differences in the *robust* SD estimation, I assumed that that epsilon is tolerable.